services/brig/test/integration/API/Team.hs

```diff
@@ -159,6 +160,30 @@ testCreateTeam brig galley = do
     team3 <- getTeam galley (team^.Team.teamId)
     liftIO $ assertEqual "status" Team.Active (Team.tdStatus team3)
 
+testCreateTeamPreverified :: Brig -> Galley -> Http ()
+testCreateTeamPreverified brig galley = do
+    email <- randomEmail
+    requestActivationCode brig (Left email)
+    act <- getActivationCode brig (Left email)
+    case act of
+        Nothing     -> liftIO $ assertFailure "activation key/code not found"
+        Just (_, c) -> do
+            rsp <- register' email newTeam c brig <!! const 201 === statusCode
+            let Just uid = userId <$> decodeBody rsp
+            teams <- view Team.teamListTeams <$> getTeams uid galley
+            liftIO $ assertBool "User not part of exactly one team" (length teams == 1)
+            let team = fromMaybe (error "No team??") $ listToMaybe teams
+            liftIO $ assertBool "Team not binding" (team^.Team.teamBinding == Team.Binding)
+            mem <- getTeamMember uid (team^.Team.teamId) galley
+            liftIO $ assertBool "Member not part of the team" (uid == mem ^. Team.userId)
+            team2 <- getTeam galley (team^.Team.teamId)
+            -- Team should already be active
+            liftIO $ assertEqual "status" Team.Active (Team.tdStatus team2)
```

Comment:
"status" -> "team should already be active" and optionally remove comment above

Reply:
👍